examples/identity/src/main.sw

```diff
@@ -25,7 +25,7 @@ storage {
 impl IdentityExample for Contract {
     fn cast_to_identity() {
         // ANCHOR: cast_to_identity
-        let my_address: Address = ~Address::from(BASE_ASSET_ID);
+        let my_address: Address = ~Address::from(BASE_ASSET_ID.into());
```

Comment:
Hmmm. This doesn't really make sense now since you don't want to be using an asset ID as an address. Can you change to something like
```suggestion
        let raw_address: b256 = 0xddec0e7e6a9a4a4e3e57d08d080d71a299c628a46bc609aab4627695679421ca;
        let my_address: Address = ~Address::from(raw_address);
```

Reply:
I did think that the `.into()` was a bit magical here in the conversion but thought that the next issue where a `const b256` would just update this after

Reply:
Done